Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN groupadd -g 999 cnc_group && \
     chown cnc_user:cnc_group /home/cnc_user/.pan_cnc
 
 ADD requirements.txt /app/requirements.txt
+ADD tox.ini /app/tox.ini
+ADD .gitmodules /app/.gitmodules
+ADD .git /app/.git
```

Comment:
all this should be present in the image already ?

General strategy is that we want to avoid changing the 'production' image just for testing. Def can see that we need to add the tests dir and tox.ini though, so those changes are fine